testbed/tests/log_test.go

```diff
@@ -263,6 +264,10 @@ func TestLogLargeFiles(t *testing.T) {
 				ItemsPerBatch:      1,
 				Parallel:           100,
 			},
+			resourceSpec: testbed.ResourceSpec{
+				ExpectedMaxCPU: 80,
+				ExpectedMaxRAM: 150,
+			},
```

Comment:
i'd say yes - at least somewhat , as I was using this test for an initial comparison between the synchronous and asynchronous approach - without specifying resource limits here the CPU/memory consumption is not recorded and included in the test summary - however this can maybe be reverted when we have a better benchmark test for the adapter, based on your suggestion